### ISSUE TITLE
mac Incompatible provider version for 1.23.1

### ISSUE THREAD

**Issue by powellquiring** [IBM-Cloud/terraform-provider-ibm#2494]
```
➜  tf tf version
Terraform v0.14.10
➜  tf cat version.tf
terraform {
  required_providers {
    ibm = {
      source = "IBM-Cloud/ibm"
    }
  }
}
➜  tf tf init

Initializing the backend...

Initializing provider plugins...
- Finding latest version of ibm-cloud/ibm...

Error: Incompatible provider version

Provider registry.terraform.io/ibm-cloud/ibm v1.23.1 does not have a package
available for your current platform, darwin_amd64.

Provider releases are separate from Terraform CLI releases, so not all
providers are available for all platforms. Other versions of this provider may
have different platforms supported.

```

**Comment by hkantare** [comment]
Its aknow issue we are having some issue with notarization..once notarization process is fixed we will do an immediate release

**Comment by kavya498** [comment]
New release is available [1.23.2](https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v1.23.2)
closing this issue.. thank you..
